src/library/scala/math/package.scala

```diff
@@ -20,7 +20,7 @@ package scala
   * @groupprio math-const 10
   *
   * @groupname minmax Minimum and Maximum
-  * @groupdesc minmax Find the min or max of two numbers. Note: [[scala.collection.TraversableOnce]] has
+  * @groupdesc minmax Find the min or max of two numbers. Note: [[scala.collection.Iterable]] has
```

Comment:
They're defined in `IterableOnceOps`, so also available on `Iterator` - worth mentioning?